tests/unit/test_utils.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from labelbox.utils import format_iso_datetime, format_iso_from_string
+
+
+@pytest.mark.parametrize(
+    'datetime_str, expected_datetime_str',
+    [
+        ('2011-11-04T00:05:23Z', '2011-11-04T00:05:23Z'),
+        ('2011-11-04T00:05:23+00:00', '2011-11-04T00:05:23Z'),
+        ('2011-11-04T00:05:23+05:00', '2011-11-04T00:05:23Z'
+        ),  #NOTE not converting with timezone... this is compatible with out current implementation
```

Comment:
Seeing some typos around the comments. Since it's a public repository, let's make sure we don't have too many of these :)
Could you proofread all the comments?